zhit: 0.9
zshort: 0
zmax: 0.09
zrand: 0.01
sigma_hit: 0.01
lambda_short: 0.7393790027529336